src/docfx/build/metadata/ContributionProvider.cs

```diff
@@ -169,6 +169,12 @@ public void Save()
             }
         }
 
+        private PathString GetOriginalFullPath(FilePath file)
+        {
+            var originalPath = _sourceMap.GetOriginalFilePath(file.Path);
```

Comment:
With this change, should `SourceMap.GetOriginalFilePath` take `FilePath` as input to only apply to `FileOrigin.Main`?

Reply:
yes, it should be. Adding an assert for it.